Adapter/RedisTagAwareAdapter.php

```diff
@@ -97,7 +97,7 @@ protected function doSave(array $values, int $lifetime, array $addTagData = [],
         if ('noeviction' !== $eviction && 0 !== strpos($eviction, 'volatile-')) {
             CacheItem::log($this->logger, sprintf('Redis maxmemory-policy setting "%s" is *not* supported by RedisTagAwareAdapter, use "noeviction" or  "volatile-*" eviction policies', $eviction));
 
-            return false;
+            return [];
```

Comment:
for the PR on the right repo, note that this should list the *failed* ids,
so it should be `return array_keys($values);`

Reply:
maybe the best solution will be throw an exception?

Reply:
the exception would be turned into a log message (the cache layer catches them by design)